lib/nostrum/util.ex

```diff
@@ -204,6 +204,39 @@ defmodule Nostrum.Util do
       String.to_atom(token)
   end
 
+   # Generic casting function
```

Comment:
Don't think this should be indented a space.

Reply:
Good catch 👍 . Fixed in latest commit.